package.json

```diff
@@ -0,0 +1,153 @@
+{
+  "name": "trampss-todomvc",
+  "version": "0.1.0",
+  "description": "TodoMVC with React / Redux / Redux-Saga",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-dev-server --host 0.0.0.0 --inline --hot --port=3000 --history-api-fallback --watch-poll=1000",
+    "lint:js": "eslint --ext js,jsx src",
+    "lint": "npm-run-all --parallel lint:*",
+    "test": "jest",
+    "analyze": "webpack --json > bundle.analyze.json"
+  },
+  "dependencies": {
+    "babel-polyfill": "^6.20.0",
+    "isomorphic-fetch": "2.2.1",

```

Comment:
Has TodoMVC some API fetch ?
If not, you could remove this dependency

Reply:
i have remove this